### ISSUE TITLE
Function loss or training code

### ISSUE THREAD

**Issue by dluvizon** [intel-isl/MiDaS#23]
Hi!

Thanks for sharing the source code, this is a great contribution!
I have searched in the repository, but I didn't find the loss function that you have described in the paper (considering the normalization procedure for disparity maps).
Do you have plans to share it, or even to share the training code?

Thanks.

**Comment by dluvizon** [comment]
Just now, after searching in the closed issues, I found #2 . Maybe if you include the link https://gist.github.com/dvdhfnr/732c26b61a0e63a0abc8a5d769dbebd0 in the README file could prevent from people opening issues with this question again.

Thanks
